2022-07_sprint/sprint_parties/contributors.md

```diff
@@ -259,4 +259,58 @@ Callistus Ndemo
 [{fab}`linkedin-in`](https://www.linkedin.com/in/ndemo/)
 [{fab}`twitter`](https://twitter.com/callistusndemo)
 :::
+:::{grid-item}
+
+<img alt="Image for contributor template" src="../../_static/contributors/blank_person.jpg" class="rounded-circle">
+
+Andrianina David Raharijao
+
+NYC, United States
+
+New contributor
```

Comment:
I am not sure this should be added inside the personal squares cc @reshamas

Reply:
Yes, agreed.  Will remove this.